src/OrleansPSUtils/Orleans.psd1

```diff
@@ -0,0 +1,24 @@
+﻿@{
+	RootModule = 'OrleansPSUtils.dll'
+	ModuleVersion = '1.2.3'
```

Comment:
I tried to do this:

```
  <UsingTask TaskName="UpdateVersion" TaskFactory="CodeTaskFactory" AssemblyFile="$(MSBuildToolsPath)\Microsoft.Build.Tasks.v4.0.dll">
    <ParameterGroup>
      <InputFilename ParameterType="System.String" Required="true" />
      <OutputFilename ParameterType="System.String" Required="true" />
    </ParameterGroup>
    <Task>
      <Reference Include="System.Core" />
      <Using Namespace="System" />
      <Using Namespace="System.IO" />
      <Using Namespace="System.Text.RegularExpressions" />
      <Code Type="Fragment" Language="cs"><![CDATA[
				var version = File.ReadAllText(InputFilename);
				var inputText = File.ReadAllTest(OutputFilename);
				File.WriteAllText(
					OutputFilename,
					inputText.Replace("$version$", version)
					);
				
			  ]]></Code>
    </Task>
  </UsingTask>
```

and in the end of the .csproj this: 

```
<Target Name="AfterBuild">
    <ReplaceFileText InputFilename="$(SolutionDir)Build\Version.txt" OutputFilename="$(OutputDir)Orleans.psd1" />
  </Target>
```

I'm not good at MSBuild so if you can put something together for it I can quickly add it here.

Thanks